apps/native/app/src/stores/auth-store.ts

```diff
@@ -136,6 +137,7 @@ export const authStore = create<AuthStore>((set, get) => ({
     return false
   },
   async logout() {
+    apolloMKKVStorage.clearStore()
     const appAuthConfig = getAppAuthConfig()
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     const tokenToRevoke = get().authorizeResult!.accessToken!
```

Comment:
@eirikurn the ticket in notion talked about making sure that Zustand state is reset. The zustand state for authStore is reset here a few lines below (see [here](https://github.com/island-is/island.is/blob/5d48548fa8d670eb53dca1df6923a9e86d68e8bf/apps/native/app/src/stores/auth-store.ts#L157) where userInfo and authorizeResult are set to undefined. Is the notion ticket talking about offline-store maybe? I don't see that that is currently reset anywhere in the code on logout - is that what you were talking about? Sorry don't have a lot of context here 😄

Reply:
Good question. Clearing mmkv + apollo cache should handle the offline store. @snaerseljan was there anything else? 🤔